server/controllers/requests.js

```diff
@@ -34,11 +32,11 @@ exports.userRequests = asycnHandler(async (req, res, next) => {
 // @route PATCH /request/:id
 // Edit a request with matching userId
 exports.editRequest = asycnHandler(async (req, res, next) => {
-  const id = req.params.id;
-  const request = req.body;
+  const id = req.body.request._id;
+  const request = req.body.request;
```

Comment:
There seems to be overuse of the name _request_, try naming variables meaningful and distinguishable if that is scope of this PR

Reply:
I'll try to make some changes, thanks for looking over it :)